src/components/Home.js

```diff
@@ -20,20 +24,26 @@ class Home extends Component {
         }
         return (
             <div className='container home'>
-                <h1>Huddle: Find a club that matches you</h1>
-                <p>
-                    Find a club, get involved, and make new friends!
-                    Find a club in less time.
-                </p>
 
-                <img src={LargeLogo} />
-                <br />
+                <Row>
+
```

Comment:
remove extra line here, creates unnecessary spacing, which is harder to see code that might be below